.gitignore

```diff
@@ -19,3 +19,4 @@ wp-cli.yml
 [Dd]esktop.ini
 *.DS_store
 .DS_store?
+.lando.local.yml
```

Comment:
Wouldn't it be better to rename our `.lando.yml` to `.lando.base.yml` and then ignore `.lando.yml` as well? This way we set the defaults, allow other setups (e.g. a project-specific shared configuration) to extend our config via `.lando.yml`, plus machine-specific adjustments via `.lando.local.yml`. I think there are three layers of configuration here, and we should probably provide the lowest. See https://thinktandem.io/blog/2019/02/01/lando-is-ready-for-the-masses-with-rc2-release/#base-file

Reply:
Yeah, that sounds fine to me.

Should we not do something similar for WP-CLI? Why not move `config/wp-cli.yml` up to `wp-cli.yml` and stop copying it. Users can create system overrides via `wp-cli.local.yml`.

I sometimes get confused which file I should be modifying since there are duplicates of the same name. I think we can make it simpler by stop copying.

Similarly, we could avoid copying `wp-config.php` and instead use a symlink.